api/openapi.yaml

```diff
@@ -637,14 +647,76 @@ paths:
           description: The service encountered an error when gathering health information
         '503':
           description: The service is not taking HTTP requests
+
+  '/bmc/bios/{xname}/TpmState':
+    get:
+      tags:
+        - bios
+      summary: Fetch the current BIOS setting for the TPM State.
+      description: >-
+        Fetch the current BIOS setting for the TPM State.
+      parameters:
+        - name: xname
+          in: path
+          description: Locational xname of BMC.
+          required: true
+          schema:
+            $ref: '#/components/schemas/xname_for_node'
+      responses:
+        '200':
+          description: OK.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bmc_bios_tpm_state'
+        '400':
+          description: Bad request
+        '404':
+          description: Xname was not for a bmc
+        '500':
+          description: Internal server error including failures communicating with the server
+    patch:
+      tags:
+        - bios
+      summary: >-
+        Set the TPM State field in the BIOS settings
+      description: >-
+        Set the TPM State in the BIOS settings.
+      parameters:
+        - name: xname
+          in: path
+          description: Locational xname of the BMC.
+          required: true
+          schema:
+            $ref: '#/components/schemas/xname_for_node'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/bmc_bios_tpm_state_put'
+      responses:
+        '204':
+          description: Value set
+        '400':
```

Comment:
use the Problem7807 body for these... 
If we cannot PAtch TpmState then return 400 and tell them the model isnt supported.